File: CrEST/Microsoft.CrEST.UI/Microsoft.CrEST.UI/angularCrest/app/sla/shared/models/applicationMetaData.ts
export class ApplicationMetaData {
    public contractIds: Array <number>;
    public suppliers: Array<Supplier>;
    public itOrg: Array<ITOrg>;
    public serviceClass: Array<ServiceClass>;
};

export class Supplier {
    public supplierId: number;
    public supplierName: string;
    public isCrest: boolean;
}


export class ITOrg {
    public itOrgId: number;
    public itOrgName: string;
    
}
export class ServiceClass {
    public id: number;
    public serviceClassName: string;
}

File: CrEST/Microsoft.Crest.Web.API/CrEST.Models/ApplicationMetadata.cs
using CrEST.Data.Models;
using System.Collections.Generic;

namespace CrEST.Models
{
    public class ApplicationMetadata
    {
        public List<Supplier> Suppliers { get; set; }
        public List<int> ContractIds { get; set; }
        public List<Itorg> ItOrg { get; set; }
        public List<ServiceClass> ServiceClass { get; set; }
    }
}
